drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/test/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/XLSBuilderCarsInCityTest.java

```diff
@@ -37,17 +36,12 @@
 public class XLSBuilderCarsInCityTest
         extends TestBase {
 
-    private static String oldDateFormatValue;
     private static String oldLanguageValue;
 
     @BeforeClass
     public static void setUpBefore() throws Exception {
-
-        oldDateFormatValue = System.getProperty("drools.dateformat");
         oldLanguageValue = System.getProperty("drools.defaultlanguage");
```

Comment:
@Rikkola In the case of Parallel testing, this could affect other tests running in parallel. I think we could consider mocking that static method, considering current mockito version 3.6.0 supports it natively. This should garantee testing isolation, WDYT?